common/build.gradle

```diff
@@ -25,4 +25,5 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.3.1'
     testCompile 'junit:junit:4.12'
+    compile 'com.google.code.gson:gson:2.8.1'
```

Comment:
We'll also want to exclude `gson` as a transitive dependency in ADAL - I've done this [here](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/9674a1d08353a9fbcec535fff6c9cab0807025c5/adal/build.gradle#L103-L105).

Reply:
Thanks!